contracts/orchestrator-contracts/tsconfig.json

```diff
@@ -1,10 +1,13 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.options.json",
   "compilerOptions": {
-    "baseUrl": "src",
-    "composite": true,
-    "outDir": "./dist/types"
+    "baseUrl": "src"
   },
+  "references": [
+    {
+      "path": "../../packages/serverless-contracts/tsconfig.build.json"
+    }
+  ],
```

Comment:
is this still needed ? 
it is already specified in the root `tsconfig.options.json`